Everything and Anything

DutchPlanDerLindeSus opened the pull request:
Had to create another branch cuz the last one was bugged and wouldnt let me open a PR, this should be the same code cuz i just merged it over from Maps-March

Adds a new/overhauled focus tree for the following nations:
- Saudi Arabia
- Bhutan
- Tibet (as of 29.4.2024 not yet done)
- Indonesia
- Philippines
The nations above also received general additions, such as GFX etc.

- Majapahit formable has been replaced with the "Pan-Malayan Confederation", the Polynesian Formable has been expanded and made realistic (forming polynesia actually requires polynesia now)

- Global Resource Rebalancing: done using multiple credible and historical sources, for more information go to my dev-chat

- Global Population Rebalancing: nothing pretty major, the amount of added/removed population doesnt exceed a million, exception: Philippines

Map Changes:
- Adds Southern Sudetenland
- Tonga and Easter Island have been added, general changes to the pacific theater in terms of historical ownership of states etc.
- Changes to the Ma Clique states to fit both Greater Mongolia and Greater Tibet
- Jafurah (actual recognized border between the Saudis and British although the Saudis later established control
- Channel Islands (UK) province as well as some other minor province tweaks, such as downsizing of some islands etc.

Notable Misc:
- Nepal is no longer a british protectorate as it wasn't historically, however, an event has been made to represent Nepal joining the war if it accepts and everything goes historically
- Brit added some stuff to Danzig as well and some other general stuff, like the impassible stuff in Borneo etc.
- Multiple bugs have been fixed as well

TheBrit7 committed:
Revert "Revert "Easter Island""

This reverts commit 74cd390130a8220edb364df1a56610daf2511021.

TheBrit7 committed:
Merge branch 'Beta-Edits' into Maps-March

TheBrit7 committed:
Tonga

TheBrit7 committed:
Swiss population fix

TheBrit7 committed:
This should of been removed already

TheBrit7 committed:
Minor Danzig expansion

DutchPlanDerLindeSus committed:
Tonga stuff

DutchPlanDerLindeSus committed:
Merge branch 'Maps-March' of https://github.com/deliciousmods/1956_beta into Maps-March

DutchPlanDerLindeSus committed:
ITS TONGA TIME

DutchPlanDerLindeSus committed:
some additions

DutchPlanDerLindeSus committed:
big stuff

DutchPlanDerLindeSus committed:
Iwo Jima

DutchPlanDerLindeSus committed:
Otranto Strait

DutchPlanDerLindeSus committed:
some stuff

DutchPlanDerLindeSus committed:
final

DutchPlanDerLindeSus committed:
Hungary accurate population

DutchPlanDerLindeSus committed:
Greece accurate population

DutchPlanDerLindeSus committed:
Albania population

DutchPlanDerLindeSus committed:
Slovakia

DutchPlanDerLindeSus committed:
Update 973-Andorra.txt

DutchPlanDerLindeSus committed:
belgium

TheBrit7 committed:
African oil and Belize

DutchPlanDerLindeSus committed:
Fixes and Tonga

DutchPlanDerLindeSus committed:
more tonga

DutchPlanDerLindeSus committed:
Polynesia formable

DutchPlanDerLindeSus committed:
POPULATIONS

DutchPlanDerLindeSus committed:
small fix

DutchPlanDerLindeSus committed:
iraq

DutchPlanDerLindeSus committed:
Saudi Arabia - Population and Borders

DutchPlanDerLindeSus committed:
OIL!!!!!

DutchPlanDerLindeSus committed:
tibet - might remove

DutchPlanDerLindeSus committed:
more tibet stuff - might remove

DutchPlanDerLindeSus committed:
more tibet stuff

DutchPlanDerLindeSus committed:
Saudi and Estonia

DutchPlanDerLindeSus committed:
est

DutchPlanDerLindeSus committed:
techs and tibet

DutchPlanDerLindeSus committed:
Saudi Arabia tweeks

DutchPlanDerLindeSus committed:
Removing unused stuff

DutchPlanDerLindeSus committed:
bhutan overhaul

DutchPlanDerLindeSus committed:
Merge remote-tracking branch 'upstream/Beta-Edits' into Maps-March

TheBrit7 committed:
Merge branch 'Beta-Edits' into Maps-March

DutchPlanDerLindeSus committed:
big changes

DutchPlanDerLindeSus committed:
BIG CHANGE

DutchPlanDerLindeSus committed:
stuff

DutchPlanDerLindeSus committed:
more

DutchPlanDerLindeSus committed:
Merge remote-tracking branch 'upstream/Beta-Edits' into Maps-March

Commit:
Merge branch 'Beta-Edits' into Maps-March

DutchPlanDerLindeSus committed:
biggie smalls

DutchPlanDerLindeSus committed:
stuff

DutchPlanDerLindeSus committed:
Update AFG.txt

Commit:
Merge branch 'Maps-March' of https://github.com/deliciousmods/1956_beta into Maps-March

Commit:
Borneo

Commit:
Texas

Commit:
Venezuela bug fix

DutchPlanDerLindeSus committed:
Massive dump

DutchPlanDerLindeSus committed:
Merge branch 'Maps-March' of https://github.com/deliciousmods/1956_beta into Maps-March

DutchPlanDerLindeSus committed:
Delete BUL_1936_naval_mtg.txt

Commit:
Portraits

Commit:
Merge branch 'Maps-March' of https://github.com/deliciousmods/1956_beta into Maps-March

SwedishNorwegian committed:
Revert "Delete BUL_1936_naval_mtg.txt"

This reverts commit ab82e28028cfbb52252f2a7f277743f0e3e48991.

SwedishNorwegian committed:
Greater Ukraine

SwedishNorwegian committed:
Update UKR.txt

DutchPlanDerLindeSus committed:
GLOBAL STEEL PRODUCTION

DutchPlanDerLindeSus committed:
Merge branch 'Maps-March' of https://github.com/deliciousmods/1956_beta into Maps-March

DutchPlanDerLindeSus committed:
Burma historical

DutchPlanDerLindeSus committed:
TUNGSTEN, CHROMIUM AND ALUMINIUM

DutchPlanDerLindeSus committed:
RUBBER

DutchPlanDerLindeSus committed:
OIL AND SOME OTHER FIXES

DutchPlanDerLindeSus committed:
Update 7-Holland.txt

DutchPlanDerLindeSus committed:
Last touches

DutchPlanDerLindeSus committed:
stuff

DutchPlanDerLindeSus committed:
Bish Bash Bosh

TheycallmeBrick committed:
Sudetenland initial

Split Sudetenland into two states.

TODO:
* Logic checks so that CZE will not surrender South Sudetenland to Germany if it would result in bordergore.
* Dynamic state names.
* More playtesting.

DutchPlanDerLindeSus committed:
Philippines done, might return to do loc

DutchPlanDerLindeSus committed:
Merge branch 'Maps-March' of https://github.com/deliciousmods/1956_beta into Maps-March

DutchPlanDerLindeSus committed:
one small fix

marijn211 committed:
Merge branch 'Beta-Edits' into Maps-March

DutchPlanDerLindeSus committed:
more fixes

DutchPlanDerLindeSus committed:
Merge branch 'Maps-March' of https://github.com/deliciousmods/1956_beta into Maps-March

TheycallmeBrick committed:
Sudetenland II

Commit:
Restoring the channel islands

Commit:
Update adjacencies.csv

DutchPlanDerLindeSus committed:
massive dump

DutchPlanDerLindeSus committed:
Merge branch 'Maps-March' of https://github.com/deliciousmods/1956_beta into Maps-March

TheycallmeBrick committed:
Sudetenland III

That should about do it.

TheycallmeBrick committed:
hollup

that's not the word, eh?

DutchPlanDerLindeSus committed:
some more stuff

DutchPlanDerLindeSus committed:
Merge branch 'Maps-March' of https://github.com/deliciousmods/1956_beta into Maps-March

DutchPlanDerLindeSus committed:
Merge remote-tracking branch 'upstream/Beta-Edits' into Maps-March

marijn211 commented:
[18:40:07][no_game_date][effectimplementation.cpp:13905]: Invalid Decision ID for unlock_decision_tooltip TIB_search_dalai_lama_in_secret

marijn211 reviewed (dismissed):
As Dutch indicated, Tibet is not finished (not just that error)

TheycallmeBrick committed:
Reversion

DutchPlanDerLindeSus committed:
some changes

DutchPlanDerLindeSus committed:
tibet done

DutchPlanDerLindeSus committed:
last change, should be good

Commit:
Texas additions

marijn211 committed:
Merge branch 'Beta-Edits' into Maps-May

marijn211 committed:
Presumed mistake by Norman on bugfix

marijn211 committed:
Update generic_advisors.txt

BiccyThiccy committed:
Italy focus fix: "Aid for the Spanish Republic"

The old system didn't let you send volunteers in almost any circumstance, which doesn't really make any sense.

What I've added just makes use of the relation-rule-override system.

Done with permission from Norrman.
